src/Compiler/Checking/CheckDeclarations.fs

```diff
@@ -744,23 +746,29 @@ module MutRecBindingChecking =
     /// Represents one element in a type definition, after the first phase    
     type TyconBindingPhase2A =
       /// An entry corresponding to the definition of the implicit constructor for a class
-      | Phase2AIncrClassCtor of IncrClassCtorLhs
+      | Phase2AIncrClassCtor of StaticCtorInfo * IncrClassCtorInfo option
```

Comment:
Why does it now contain static constructor info as well now?
Either comment should be fixed, or new union constructor introduced.

Reply:
Before this PR, info about static cctor used to be part of IncrClassCtorInfo, since those were the only types where static ctor actions were allowed.
This PR splits them into two.

=> code is correct, comment is wrong, fixing comment right now.